packages/xstate-fsm/src/index.ts

```diff
@@ -231,7 +236,13 @@ export function interpret<
         unsubscribe: () => listeners.delete(listener)
       };
     },
-    start: () => {
+    start: (initialState: TState['value']) => {
+      if (
+        initialState &&
+        Array.from(Object.keys(machine.config.states)).includes(initialState)
```

Comment:
This will probably work...

```suggestion
        initialState in machine.config.states
```

Reply:
Fixed